makefiles/docker.inc.mk

```diff
@@ -67,6 +67,7 @@ export DOCKER_ENV_VARS += \
   SIZE \
   TOOLCHAIN \
   TEST_KCONFIG \
+  IOTLAB_NODE \
```

Comment:
Can we keep the alphabetical order (which as I saw it is already a bit broken...) ?